Only in PyWavelets-0.2.0: PyWavelets-0.2.0-cython-and-libm-fix.patch
diff -ur PyWavelets-0.2.0.orig/setup.py PyWavelets-0.2.0/setup.py
--- PyWavelets-0.2.0.orig/setup.py	2012-07-09 02:07:47.296451028 +0400
+++ PyWavelets-0.2.0/setup.py	2012-07-09 02:16:26.331424477 +0400
@@ -39,7 +39,7 @@
         include_dirs = ['src'],
         library_dirs = [],
         runtime_library_dirs = [],
-        libraries = [],
+        libraries = ['m'],
         define_macros = macros,
         extra_compile_args = extra_compile_args,
         extra_link_args = [],
diff -ur PyWavelets-0.2.0.orig/src/_pywt.pyx PyWavelets-0.2.0/src/_pywt.pyx
--- PyWavelets-0.2.0.orig/src/_pywt.pyx	2012-07-09 02:07:47.298451028 +0400
+++ PyWavelets-0.2.0/src/_pywt.pyx	2012-07-09 02:09:44.741445020 +0400
@@ -164,7 +164,7 @@
 
     #cdef readonly properties
 
-    def __new__(self, char* name="", object filter_bank=None):
+    def __cinit__(self, char* name="", object filter_bank=None):
         cdef object family_code, family_number
         cdef object filters
         cdef int filter_length
